### ISSUE TITLE
Auth and refresh tokens return identical values for non-admin users

### ISSUE THREAD

**Issue by g-elwell** [wp-graphql/wp-graphql-jwt-authentication#45]
In my app I noticed that the returned auth and refresh tokens from a login mutation are returning identical values.

The root cause appears to be a capability check which prevents you from retrieving the JWT auth secret of the current user:

https://github.com/wp-graphql/wp-graphql-jwt-authentication/blob/522cb2498db379bed718da016f2530c9b38a09d7/src/Auth.php#L207-L221

This check fails if you are a regular user without edit capabilities trying to retrieve your own JWT auth secret. In turn this causes a number of filters to fail elsewhere including the one that extends the expiration date of the refresh token - this is why the returned refresh token is identical to the auth token for non-admin users.

Should the check be:
```
if ( ! $is_current_user && ! current_user_can( $capability ) ) {
```
Since you always want the current user to be able to retrieve their own JWT auth secret?

Apologies if I'm missing something, thanks for the great work on this plugin and WPGraphQL in general!

**Comment by igoojoe** [comment]
I've stuck this on a [branch in my fork](https://github.com/igoojoe/wp-graphql-jwt-authentication/tree/bug/%2345-auth-and-refresh-equal), been experiencing the same issue. Have yet to experience any other issues with this patch

**Comment by virgilbugnariu** [comment]
Sorry for bumping, but are there any plans of including this fix in a nearby release?

**Comment by jasonbahl** [comment]
I've opened a pull request with the fix for this: #64

**Comment by jasonbahl** [comment]
closed by #64
